CenterForOpenScience/cas-overlay
# [HotFix] Sign In Support For OSF Registries [CAS-59] [OSF-7753]

## Purpose

Add SignIn Support For OSF Registries.

![osf-registries-sign-in](https://cloud.githubusercontent.com/assets/3750414/24720345/e176836c-1a0a-11e7-9011-64a42d42bbf1.png)

## Changes

* Sign In page has OSF Registries banner and title
* User lands on the original page they come from after successful login
* Link "Create accounts" redirects user to the dedicated sign up page

For Sign Up Changes, see https://github.com/CenterForOpenScience/osf.io/pull/7072.

## Deployment Notes

@icereval add `registries-osf.json` to services.

## QA Notes

The tests should cover both signup [OSF-7753](https://openscience.atlassian.net/browse/OSF-7753) and login [CAS-59](https://openscience.atlassian.net/browse/CAS-59) logic.

For signup, verify that:
1. OSF Registries Banner and Title to the signup page
2. New user receives customized confirmation email
3. User lands on OSF Registries home page after confirmation

For login, verify that:
1. Login page should have OSF Registry Banner and Title
2. User lands on the page where they click the login button after successful sign in
3. Create accounts redirect user the customized sign up page

## Side effects

No

## Ticket

Sign Up Support: https://openscience.atlassian.net/browse/OSF-7753
Sing In Support: https://openscience.atlassian.net/browse/CAS-59
